components/omega/OmegaBuild.cmake

```diff
@@ -19,11 +19,17 @@ set(OMEGA_DEFAULT_BUILD_TYPE      Release) # Debug or Release
 ###########################
 macro(setup_common_variables)
  
+  option(OMEGA_DEBUG "Turn on error message throwing (default OFF)." OFF)
+
   if(NOT DEFINED OMEGA_CXX_FLAGS )
     # initialize cxx flags as an empty list
     set(OMEGA_CXX_FLAGS "")
   endif()
 
+  if(NOT DEFINED OMEGA_LOG_FILEPATH)
+    set(OMEGA_LOG_FILEPATH "${PROJECT_BINARY_DIR}/omega.log")
```

Comment:
Polaris can certainly work around this but this seems like an odd default to me.  Hopefully, we will have a way of defining a run directory in the near future, and that would then be a more appropriate default.  

Perhaps more importantly, it seems strange to define the log file name at compile time.  I would want to use the same executable for lots and lots of runs, and each run would need to have its own log file.  How would I specify a different location for the log file at *runtime*?

Reply:
I think that your concerns are legit. Setting a logfile path using OMEGA_LOG_FILEPATH is temporary solution mostly for debugging.

In e3sm-embedded build mode, we may have to collect the run/build directories and log file paths from E3SM configurations. As of this PR, I have not implemented getting the paths from E3SM yet.

In standalone build mode, we may have to collect those path info from Omega local configuration. At this time, I am sure if there exists the configuration file, so I just used the temporary(debugging) macro.

sdplog package does support creating new loggers at runtime. So, we may be able to use the spdlog feature to create a new logger and generate a new log file at runtime if needed.

Reply:
Okay, sounds good.  This will be a stopgap until with have a config file.

Reply:
Actually, should we just not include a path for this? That way the log file is just created where the executable is and we can use soft links if the log file should show up elsewhere. That's how the current MPAS works and seems to be reasonable behavior?

Reply:
Ok. I will remove the logfile path and use a hard-coded path as the current MPAS does.

Reply:
@grnydawn, I think the point is that we don't want a path at all (hard coded or otherwise).  We want the log file to be created in the directory where the executable is being run.  I.e. the path should be `./omega.log` or just `omega.log`.